db/index.js

```diff
@@ -1,36 +1,35 @@
-require('dotenv').config();
-const mongoose = require('mongoose');
+const mariadb = require('mariadb');
 
-mongoose.connect('mongodb+srv://zallo:zallo123@cluster0-s3ma8.mongodb.net/test?retryWrites=true', { useNewUrlParser: true });
-
-const houseSchema = mongoose.Schema({
-  _id: Number,
-  address: String,
-  price: Number,
-  beds: Number,
-  baths: Number,
-  floorSize: Number,
-  description: String,
-  type: String,
-  year: Number,
-  heating: String,
-  cooling: String,
-  parking: Number,
-  lotSize: Number,
-  daysListed: Number,
-  saves: Number,
-  appliances: Array,
-  interiorFeatures: Array,
-  construction: String,
-  roof: String,
-  exterior: String,
-  flooring: String,
-  rooms: Number,
-  stories: Number,
-  spaces: String,
-});
+// sock : /run/mysqld/mysqld.sock --I found this by stopping the mariaDb service and attempting to run the cli
+const homes = mariadb.createPool({socketpath: '/run/mysqld/mysqld.sock', user: 'patrick', password:'ekauq', connectionLimit: 5, database:'gosackService'});
 
 module.exports = {
-  House: mongoose.model('House', houseSchema),
-  House_test: mongoose.model('House_test', houseSchema),
-};
+     House: homes
+  };
+
+// const houseSchema = mongoose.Schema({
```

Comment:
Don't commit code comments. If you want to keep around a different version of some code, create a different commit and/or branch.